test/data/filters/html/file-HTMLFilter2-tag-dropping-bugfix-SF609.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html>
+<head><meta charset="UTF-8"><script>{}</script><style>
```

Comment:
The failing test we saw before seems to have been caused by the case of `utf-8`: OmegaT turns it into upper-case letters which caused the assert to fail. This behaviour (the change of case) existed before and is not related to the bugs as far as I can tell.